src/EditorFeatures/CSharpTest/DocumentationComments/CodeFixes/RemoveDocCommentNodeCodeFixProviderTests.cs

```diff
@@ -0,0 +1,606 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.DiagnosticComments.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.DocumentationComments.CodeFixes
+{
+    public class RemoveDocCommentNodeCodeFixProviderTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
+    {
+        internal override Tuple<DiagnosticAnalyzer, CodeFixProvider> CreateDiagnosticProviderAndFixer(Workspace workspace)
+        {
+            return new Tuple<DiagnosticAnalyzer, CodeFixProvider>(null, new CSharpRemoveDocCommentNodeCodeFixProvider());
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveDuplicateParamTag)]
+        public async Task RemovesDuplicateParamTag()
+        {
+            var initial = 
+@"class Program
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name=""value""></param>
+    /// <param [|name=""value""|]></param>
+    public void Fizz(int value) {}
+}
+";
+
+            var expected =
+@"class Program
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name=""value""></param>
+    public void Fizz(int value) {}
+}
+";
+            var parseOptions = Options.Regular.WithDocumentationMode(DocumentationMode.Diagnose);
```

Comment:
Also, the core test method you call *must* pass in:

, compareTokens: false

The reason for this is that if you don't pass this in, then our test code will explicitly *ignore* trivia.  That's obviously bad because this entire feature is precisely about changing trivia.  :)